packages/graphql/src/translate/create-connect-and-params.ts

```diff
@@ -436,7 +436,7 @@ function createConnectAndParams({
                     res.params = { ...res.params, ...subquery.params };
                 }
             });
-            res.connects.push(subqueries.join("\nUNION\n"));
+            res.connects.push(subqueries.join("\n}\nCALL {"));
```

Comment:
question (non-blocking): Why is this not also `\n}\nCALL {\n\t` like in the 'disconnect' and 'update' file?
I also see some tck tests where the `CALL` and `WITH ...` are in the same line rather than the `WITH ...` being on a new line.